Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -92,14 +142,42 @@ pathManagerState& cruisingState::getInstance()
 
 void coordinateTurnElevation::execute(pathManager* pathMgr)
 {
-    //get elevation and turning data
+    
+    SFOutput_t * pSensFusionOutput = sensorFusion::GetSFOutput(); // Get sensor fusion data
+    _WaypointManager_Data_Out * waypointOutput = cruisingState::GetOutputData(); // Get output data from waypoint manager
+    
+    // Set up the coordinated turn function input
+    double track = atan(pSensFusionOutput->latitudeSpeed / pSensFusionOutput->longitudeSpeed) * 180.0 / PI; // Gets angle of track in quadrant 1
+
+    /* 
+    
+    Need a way to convert the value above to a good track value 
+    
```

Comment:
What's the problem with directly using IMU speeds for calculating track? I have something similar in the takeoff stuff, was going to add that to sensor fusion when merging.